tests/data/test_md_report_custom_sections.md

```diff
@@ -15,98 +15,341 @@
 | ------------------| ----------- | ------------- | ------------- | ------------- | ----------- | -------------------| ------------------|
-| s1-spine1 | 30 | 4 | 9 | 15 | 2 | AVT, Field Notices, Hardware, ISIS, LANZ, OSPF, PTP, Path-Selection, Profiles | AAA, BFD, BGP, Connectivity, Cvx, Interfaces, Logging, MLAG, SNMP, STUN, Security, Services, Software, System, VLAN |
+| s1-spine1 | 181 | 43 | 34 | 103 | 1 | AVT, Field Notices, Flow Tracking, Hardware, ISIS, Interfaces, LANZ, OSPF, PTP, Path-Selection, Profiles, Segment-Routing | AAA, BFD, BGP, Configuration, Connectivity, Cvx, Greent, Interfaces, Logging, MLAG, Multicast, Routing, SNMP, STP, STUN, Security, Services, Software, VLAN, VXLAN |
 
 ### Summary Totals Per Category
 
 | Test Category | Total Tests | Tests Success | Tests Skipped | Tests Failure | Tests Error |
 | ------------- | ----------- | ------------- | ------------- | ------------- | ----------- |
-| AAA | 1 | 0 | 0 | 1 | 0 |
-| AVT | 1 | 0 | 1 | 0 | 0 |
-| BFD | 1 | 0 | 0 | 1 | 0 |
-| BGP | 1 | 0 | 0 | 0 | 1 |
-| Configuration | 1 | 1 | 0 | 0 | 0 |
-| Connectivity | 1 | 0 | 0 | 1 | 0 |
-| Cvx | 1 | 0 | 0 | 0 | 1 |
-| Field Notices | 1 | 0 | 1 | 0 | 0 |
-| Hardware | 1 | 0 | 1 | 0 | 0 |
-| Interfaces | 1 | 0 | 0 | 1 | 0 |
-| ISIS | 1 | 0 | 1 | 0 | 0 |
+| AAA | 7 | 0 | 0 | 7 | 0 |
+| AVT | 3 | 0 | 3 | 0 | 0 |
+| BFD | 4 | 1 | 0 | 3 | 0 |
+| BGP | 25 | 3 | 0 | 21 | 1 |
+| Configuration | 3 | 1 | 0 | 2 | 0 |
+| Connectivity | 2 | 1 | 0 | 1 | 0 |
+| Cvx | 5 | 0 | 0 | 5 | 0 |
+| Field Notices | 2 | 0 | 2 | 0 | 0 |
+| Flow Tracking | 1 | 0 | 1 | 0 | 0 |
+| Greent | 2 | 0 | 0 | 2 | 0 |
+| Hardware | 7 | 0 | 7 | 0 | 0 |
+| Interfaces | 16 | 7 | 1 | 8 | 0 |
+| ISIS | 7 | 0 | 7 | 0 | 0 |
 | LANZ | 1 | 0 | 1 | 0 | 0 |
-| Logging | 1 | 0 | 0 | 1 | 0 |
-| MLAG | 1 | 0 | 0 | 1 | 0 |
-| OSPF | 1 | 0 | 1 | 0 | 0 |
-| Path-Selection | 1 | 0 | 1 | 0 | 0 |
-| Profiles | 1 | 0 | 1 | 0 | 0 |
-| PTP | 1 | 0 | 1 | 0 | 0 |
-| Routing | 1 | 1 | 0 | 0 | 0 |
-| Security | 2 | 0 | 0 | 2 | 0 |
-| Services | 1 | 0 | 0 | 1 | 0 |
-| SNMP | 1 | 0 | 0 | 1 | 0 |
-| Software | 1 | 0 | 0 | 1 | 0 |
-| STP | 1 | 1 | 0 | 0 | 0 |
-| STUN | 2 | 0 | 0 | 2 | 0 |
-| System | 1 | 0 | 0 | 1 | 0 |
-| VLAN | 1 | 0 | 0 | 1 | 0 |
-| VXLAN | 1 | 1 | 0 | 0 | 0 |
+| Logging | 10 | 3 | 0 | 7 | 0 |
+| MLAG | 6 | 4 | 0 | 2 | 0 |
+| Multicast | 2 | 1 | 0 | 1 | 0 |
+| OSPF | 3 | 0 | 3 | 0 | 0 |
+| Path-Selection | 2 | 0 | 2 | 0 | 0 |
+| Profiles | 2 | 0 | 2 | 0 | 0 |
+| PTP | 5 | 0 | 5 | 0 | 0 |
+| Routing | 6 | 2 | 0 | 4 | 0 |
+| Security | 15 | 3 | 0 | 12 | 0 |
+| Segment-Routing | 3 | 0 | 3 | 0 | 0 |
+| Services | 4 | 1 | 0 | 3 | 0 |
+| SNMP | 12 | 0 | 0 | 12 | 0 |
+| Software | 3 | 1 | 0 | 2 | 0 |
+| STP | 7 | 4 | 0 | 3 | 0 |
+| STUN | 3 | 0 | 0 | 3 | 0 |
+| System | 8 | 8 | 0 | 0 | 0 |
+| VLAN | 3 | 0 | 0 | 3 | 0 |
+| VXLAN | 5 | 3 | 0 | 2 | 0 |
 
 ## Failed Test Results Summary
```

Comment:
Why is this section not part of the table of content?

Reply:
closing this but it means we don't have a nice way to show users how to do custom TOC